Frontend/src/comopnents/login/login.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import axios from "axios";
+
+const Login = ({ setToken }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleLogin = async () => {
+    try {
+      const response = await axios.post("http://localhost:3001/login", {
```

Comment:
you should use Environment Variables to store, `http://localhost:3001` later when you deploy your application, you will need to replace it with The web app domain, 

Checkout: https://www.youtube.com/watch?v=y-5fkR9vGdA